src/platform/site-wide/cta-widget/helpers.js

```diff
@@ -90,32 +116,32 @@ export const toolUrl = appId => {
   switch (appId) {
     case widgetTypes.HEALTH_RECORDS:
       return {
-        url: `${mhvBaseUrl()}/mhv-portal-web/download-my-data`,
+        url: mhvUrl('download-my-data'),
         redirect: false,
       };
 
     case widgetTypes.RX:
       return {
-        url: `${mhvBaseUrl()}/mhv-portal-web/web/myhealthevet/refill-prescriptions`,
+        url: mhvUrl('web/myhealthevet/refill-prescriptions'),
```

Comment:
TODO: 

- [ ] Verify whether we can reduce this to just 'refill-prescriptions' (page looks almost identical with a few different alerts)